### DISCUSSION TITLE
Step files

### DISCUSSION THREAD

**Discussion by megatech1966** [supermerill/SuperSlicer#3332] [Community contribution]
Will super slicer support step files in the future. Like prusaslicer.

**Comment by julianjc84** [comment]
I am very keen for SuperSlicer and Step

**Comment by emersonsc** [comment]
Is there ever going to be an answer to this...?

**Reply by TwomD** [reply]
doesn't it already support step files ?
Or maybe it's only enabled in the recent nightly built.... that's what I'm using, and step files definitely work there.

**Reply by O-beron** [reply]
> doesn't it already support step files ? Or maybe it's only enabled in the recent nightly built.... that's what I'm using, and step files definitely work there.

Yes it's only on the nightly_dev branch, it works on 2.5.59.2 but not on the 2.4.58.5 build

**Reply by vkhodygo** [reply]
@O-beron That's odd because my build throws an error saying it can't find the OCCTWrapper lib.

**Reply by ThirteenPeeps** [reply]
@vkhodygo  FWIW mine says the same (I'm on Linux)

**Reply by vkhodygo** [reply]
@ThirteenPeeps It looks like STEP file support got partially broken at some point. I tried to find corresponding commits in PrusaSlicer, and they all seem to have been merged.

**Reply by TracyPeterson** [reply]
Here to ask the same question it looks like this has been open for a while now.  Any chance someone is still looking at this?

**Comment by supermerill** [comment]
hello
For step import,  the software need the OCCTWrapper library.
I just checked and latests release for windows have it.
The linux (and i guess macos) don't have it. I have to update the build process. 
#3699
